graphics/cairomm/Portfile

```diff
@@ -1,35 +1,41 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+
 PortGroup           active_variants 1.1
+PortGroup           boost 1.0
+PortGroup           meson 1.0
 
 name                cairomm
-version             1.12.2
+version             1.16.2
```

Comment:
This version of cairomm has a new major version of the libcairomm library. All ports that link with the library will be broken after this update unless you revbump them.

Reply:
I'll add these commits after I see that the CI builds this version.

Reply:
Added. Note that `py*-graph-tool` must be updated in #17914.